packages/@aws-cdk/aws-stepfunctions-tasks/test/databrew/start-job-run.test.ts

```diff
@@ -0,0 +1,70 @@
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import { DataBrewStartJobRun } from '../../lib/databrew/start-job-run';
+
+describe('Start Job Run', () => {
+
+  test('default settings', () => {
+    // GIVEN
+    const stack = new cdk.Stack();
+
+    // WHEN
+    const task = new DataBrewStartJobRun(stack, 'JobRun', {
+      name: 'jobName',
+    });
+
+    // THEN
+    expect(stack.resolve(task.toStateJson())).toEqual({
+      Type: 'Task',
+      Resource: {
+        'Fn::Join': [
+          '',
+          [
+            'arn:',
+            {
+              Ref: 'AWS::Partition',
+            },
+            ':states:::databrew:startJobRun',
+          ],
+        ],
+      },
+      End: true,
+      Parameters: {
+        Name: 'jobName',
+      },
+    });
+  });
+
+  test('sync integrationPattern', () => {
```

Comment:
missing a test for invalid integration pattern (i.e. supplying callback / `WAIT_FOR_TASK_TOKEN`)

Reply:
`WAIT_FOR_TASK_TOKEN` pattern is not supported.

Reply:
I understand, the ask is to write a test to ensure that an exception is thrown when wait for task token is requested. The other service integrations should have some inspiration for what this test should look like.